server-function/src/getdeps.ts

```diff
@@ -0,0 +1,26 @@
+import fs from 'fs';
+const contents = JSON.parse(fs.readFileSync('package-lock.json', 'utf8'));
```

Comment:
This fails if user does not have `package-lock.json` (I didn't), should we support this flow?

Reply:
We can add a warning for several cases:

- user running the command outside of a CRA project (or a subdirectory).
- user chose to use yarn (and we don't support it yet).
- user manually removed package-lock.json and needs to run `npm install` before.

Reply:
This should be raised as an issue also linked to #104